docs/dev/test-configuration-options.md

```diff
@@ -603,6 +603,12 @@ Enables the use of the app's private app container directory instead of the shar
 
 Enables mobile app instrumentation (Android or iOS) and recording of HTTP/HTTPS network traffic for debugging purposes. API calls are collected into a HAR file, which you can view and download from your **Test Results** > **Network** tab console. The default value is `false`.
 
+---
+### `audioCapture`
```

Comment:
@kristofmuhi , is there a capability for Espresso/XCUITest as well?

Reply:
the Espesso/XCUITest name will be the same just like for networkCapture! 
LESS CONFUSION! (finally)

Reply:
@aoleszyn Per @kristofmuhi 's clarification, please add the `networkCapture` capability to the [Espresso](https://docs.saucelabs.com/mobile-apps/automated-testing/espresso-xcuitest/espresso/) and [XCUITest](https://docs.saucelabs.com/mobile-apps/automated-testing/espresso-xcuitest/xcuitest/) docs.

Reply:
@kimsauce  lets do that in an other PR, since the backend is not yet ready with XCUI and Espresso